src/one_dragon/base/operation/operation_notify.py

```
@@ -107,13 +109,21 @@ def send_application_notify(app: Application, status: bool | None) -> None:
+    # 运行时间
+    duration_text = ''
+    if status is not None:
+        cost = time.time() - app.operation_start_time
+        m = int(cost // 60)
+        s = int(cost % 60)
+        duration_text = f"\n{gt('总耗时')}「{m}{gt('分')}{s}{gt('秒')}」"
+
     # 构建消息
     try:
         app_name = app.ctx.run_context.get_application_name(app_id)
     except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)